docs/overview.rst

```diff
@@ -15,7 +15,7 @@ Optionally, your journal can be encrypted using the `256-bit AES <http://en.wiki
 Why keep a journal?
 -------------------
 
-Journals aren't only for 13-year old girls and people who have too much time on their summer vacation. A journal helps you to keep track of the things you get done and how you did them. Your imagination may be limitless, but your memory isn't.
+Journals aren't just for angsty teenagers and people who have too much time on their summer vacation. A journal helps you to keep track of the things you get done and how you did them. Your imagination may be limitless, but your memory isn't.
```

Comment:
s/Journals aren't just for angsty teenagers and people who have too much time on their summer vacation/Journals are for everyone/

Reply:
I like this much better